jwt/api_jwt.py

```diff
@@ -150,7 +150,7 @@ def decode(
         detached_payload: Optional[bytes] = None,
         # passthrough arguments to _validate_claims
         # consider putting in options
-        audience: Optional[str] = None,
+        audience: Optional[Union[str, Iterable]] = None,
```

Comment:
Hey @auvipy, to be consistent with the suggestion applied L79, we can also align this one :) 

```suggestion
        audience: Optional[Union[str, Iterable[str]]] = None,
```

Reply:
Ah. According to the CI, python 3.7 does not like it. 

Maybe we can use `from __future__ import annotations` ?

```
Python 3.7.13
>>> from collections.abc import Iterable
>>> x: Iterable[str] = ["x"]
Traceback (most recent call last):
  File "<stdin>", line 1, in <module>
TypeError: 'ABCMeta' object is not subscriptable
>>> from __future__ import annotations
>>> x: Iterable[str] = ["x"]
>>>

```

Reply:
Done (commits squashed)  :heavy_check_mark: